### PR TITLE
feat: rocketchat notification

### PR DESCRIPTION
**Pull request by bryce-mcmath**
Making use of https://github.com/RocketChat/Rocket.Chat.GitHub.Action.Notification

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -235,6 +235,19 @@ jobs:
           if-no-files-found: error
           retention-days: 7
 
+      - name: Rocket.Chat iOS Notification
+        if: failure() && github.ref_name == 'main' && needs.check-ios-secrets.outputs.isReleaseBuild == 'true'
+        uses: RocketChat/Rocket.Chat.GitHub.Action.Notification@master
+        with:
+          type: ${{ job.status }}
+          job_name: 'BC Wallet iOS Build - Run number ${{ github.run_number }}'
+          mention: 'jason.leach @cvarjao @wade.king @bryce.mcmath'
```
Comment (jleach) [inline .github/workflows/main.yaml]: Do I need an`@` in my name?

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -344,6 +357,19 @@ jobs:
           if-no-files-found: error
           retention-days: 7
 
+      - name: Rocket.Chat Android Notification
+        if: failure() && github.ref_name == 'main' && needs.check-android-secrets.outputs.isReleaseBuild == 'true'
+        uses: RocketChat/Rocket.Chat.GitHub.Action.Notification@master
+        with:
+          type: ${{ job.status }}
+          job_name: 'BC Wallet Android Build - Run number ${{ github.run_number }}'
+          mention: 'jason.leach @cvarjao @wade.king @bryce.mcmath'
```
Comment (jleach) [inline .github/workflows/main.yaml]: Same.

**Review by jleach — CHANGES_REQUESTED** [review]
One change on my name. Since its the same YAML in 2x places consider making it a separate action like we do for installing node.

File: .github/workflows/main.yaml (hunk 1 above)
Comment (bryce-mcmath) [reply .github/workflows/main.yaml]: See here how mention is used: https://github.com/RocketChat/Rocket.Chat.GitHub.Action.Notification/blob/2966f5f2e0313a94f965a51c08700cfecf0b6572/src/rocketchat.ts#L120

It's just string interpolation, an '@' followed by whatever you put as `mention`. So the first name doesn't need an @. Not pretty but that's how they implemented it

File: .github/workflows/main.yaml (hunk 2 above)
Comment (bryce-mcmath) [reply .github/workflows/main.yaml]: See my other comment

**Review by jleach — APPROVED** [review]

**Pull request merged**